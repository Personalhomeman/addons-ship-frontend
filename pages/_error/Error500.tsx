import React from 'react';
import { Flex, Status500 } from '@bitrise/bitkit';

export default () => (
  <Flex direction="vertical" gap="x10" grow alignChildrenVertical="middle">
    <Status500 />
  </Flex>
);
