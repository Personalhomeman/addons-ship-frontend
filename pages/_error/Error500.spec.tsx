import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Error500 from './Error500';

describe('Error500', () => {
  it('renders ok', () => {
    const tree = shallowToJson(shallow(<Error500 />));
    expect(tree).toMatchSnapshot();
  });
});
