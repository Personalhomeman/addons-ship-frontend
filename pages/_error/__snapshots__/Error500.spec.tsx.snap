// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error500 renders ok 1`] = `
<Flex
  alignChildrenVertical="middle"
  direction="vertical"
  gap="x10"
  grow={true}
>
  <Status500 />
</Flex>
`;
